### ISSUE TITLE
MacOS arm binary fails to launch

### ISSUE THREAD

**Issue by cosmikwolf** [hacknus/serial-monitor-rust#107]
<img width="372" alt="image" src="https://github.com/user-attachments/assets/81f2fb5e-819a-4ca2-b5b1-5a5c7df3080d">

I downloaded the latest pre-compiled arm binary:
https://github.com/hacknus/serial-monitor-rust/releases/tag/v0.3.0

**Comment by hacknus** [comment]
Yes, I have the same issue. Have you tried what is specified in the `README.MD`:

> Running the apple silicon binary may result to the message "Serial Monitor is damaged and cannot be opened.", to get around this you first need to run:
> ```shell
> xattr -rd com.apple.quarantine Serial\ Monitor.app
> ```

otherwise: running the intel version should also be completely fine.

**Comment by hacknus** [comment]
@cosmikwolf were you able to solve it like this?
